release-config.yml

```diff
@@ -0,0 +1,48 @@
+project:
+  name: example-project
+  description: "Example project for publishing to PyPI"
+publisher:
+  name: providers
+  url: "https://dist.apache.org/repos/dist/dev/airflow"
+  path: "providers/"
+checks:
+  svn:
+    - id: extension
+      description: "Validate svn package extensions"
+      identifiers:
+        - type: regex
+          pattern: ".*(py3-none-any.whl|tar.gz.sha512|tar.gz.asc|tar.gz|py3-none-any.whl.asc|py3-none-any.whl.sha512)$"
+
+    - id: package_name
+      description: "Validate svn package names"
+      identifiers:
+        - type: regex
+          pattern: ".*(apache_airflow.*)$"
+
+        - type: regex
+          pattern: ".*(apache-airflow.*)$"
+
+  checksum:
+    - id: checksum
+      description: "Validate check sum with SHA512"
+      algorithm: "sha512"
+
+  signature:
+    - id: signature
+      description: "Validate signatures with GPG of packages"
+      method: gpg
+      keys: "https://dist.apache.org/repos/dist/release/airflow/KEYS"
+
+  publish:
+    id: publish
+    description: "Publish provider packages to PyPI"
+    release-type: "RC_VERSION"
+    exclude_extensions:
+      - type: regex
+        pattern: ".*(.asc|.sha512)$"
+    compare:
```

Comment:
This config for comparing the release svn location. basically for this logic to workout compare files between dev folder and release folder and copy matched files from release folder to final dist/artifact
```
for file in "${SOURCE_DIR}"/*
do
 base_file=$(basename ${file})
 cp -v "${file}" "${AIRFLOW_REPO_ROOT}/dist/${base_file//rc[0-9]/}"
 svn mv "${file}" "${base_file//rc[0-9]/}"
done
```